### ISSUE TITLE
Arctos: invalid agent_roles

### ISSUE THREAD

**Issue by MortenHofft** [gbif/model-material#120]
@tucotuco and @dustymc
The agent _role table still looks wrong to me. I'm simply looking at the values and notice that the `agent_role_target_id` columns points to IDs that look like agents. E.g. `https://arctos.database.museum/agent/21257863`. 

I believe the intention is to point to the ID of the`ENTITY` this is about. So if the type says ENTITY, then you should provide an entityID.

first row
```
agent_role_target_id,agent_role_target_type,agent_role_agent_id,agent_role_agent_name,agent_role_role,agent_role_began,agent_role_ended,agent_role_order
https://arctos.database.museum/agent/21257863,ENTITY,https://arctos.database.museum/agent/21257863,,collector,,,1
```

Related to https://github.com/gbif/model-material/issues/106

**Comment by tucotuco** [comment]
@MortenHofft That is correct. The agent_role_target_id is a foreign key to the thing the Agent had a role with respect to. The agent foreign key is supposed to go into agent_role_agent_id.
